pkg/oc/cli/admin/policy/modify_roles.go

```diff
@@ -123,7 +124,7 @@ func NewCmdAddRoleToGroup(name, fullName string, f kcmdutil.Factory, out, errOut
 			}
 
 			if len(options.Output) == 0 {
-				printSuccessForCommand(options.RoleName, true, "group", options.Targets, true, options.DryRun, out)
+				printSuccessForCommand(options.RoleName, true, "group", options.Targets, true, options.DryRun, streams.Out)
```

Comment:
:/ this does not look right - add a todo to get rid of this function later on. We should be using printFlags for success output